ophyd/signal.py

```diff
@@ -287,20 +287,20 @@ def set_thread():
             except TimeoutError:
                 success = False
                 self.log.warning(
-                    'set_and_wait(value=%s, timeout=%s, atol=%s, rtol=%s)',
-                    value, timeout, self.tolerance, self.rtolerance
+                    'set_and_wait(name=%s, value=%s, timeout=%s, atol=%s, rtol=%s)',
```

Comment:
:+1:

Reply:
one concern is that I don't think we can actually pass `name` to `set_and_wait`, but I like the idea.

Reply:
In _this particular_ case, `self.name` is accessible since the log comes from an inner (?what's the right name for this type of thingy?) function and has access to all its parent's attributes.

Reply:
Sorry, I meant that `set_and_wait(name='xyz', ..)` won't work (but it does look nice in the log!)

Would `'%s: set_and_wait(...)'` be better?